test/UserActivity-test.js

```diff
@@ -1,15 +1,34 @@
 import { expect } from "chai";
+import User from "../src/User.js";
 import UserActivity from "../src/UserActivity";
-const userActivityTestData = require("../src/data/userActivityTestData")
+const userActivityTestData = require("../src/data/userActivityTestData");
+const userData = require("../src/data/UserTestData");
 
-describe("UserActivity", ()=> {
-    let userActivity;
-    beforeEach(()=>{
-        userActivity = new UserActivity(userActivityTestData.filter(data => data.userID === 1))
-    })
-    
-    it.only('should return the miles a user has walked based on their number of steps specified by a date', ()=> {
-      const result = userActivity.numOfSteps("2019/06/15")
-        expect(result).to.equal(3577)
-      })
-})
\ No newline at end of file
+describe("UserActivity", () => {
+  let userActivity;
+  let user;
+  let userTestData;
+  beforeEach(() => {
+    userActivity = new UserActivity(
+      userActivityTestData.filter((data) => data.userID === 1)
+    );
+    user = new User(userData[0]);
+  });
+
+  it("should return the miles a user has walked based on their number of steps specified by a date (using the strideLength)", () => {
+    const result = userActivity.milesBasedOnSteps("2019/06/15", user);
```

Comment:
Based on the arguments, is the user requesting data from a particular day also needs to provide the second argument?

- I would like to add that I think we can have a method in the User class that is spicily only run when the user is requesting the data by day. so that only the date is being passed in as an argument.

Reply:
The user has all the data for one user and just has the ID and strideLength and info about the user. There is no date associated with the user just a date associated with the sleep, hydration and activity data.